frontend/src/app/modules/login/login/login.component.ts

```diff
@@ -47,7 +48,11 @@ export class LoginComponent implements OnInit {
     if (this.config.CAS_PUBLIC.CAS_AUTHENTIFICATION) {
       // if token not here here, redirection to CAS login page
       const url_redirection_cas = `${this.config.CAS_PUBLIC.CAS_URL_LOGIN}?service=${this.config.API_ENDPOINT}/gn_auth/login_cas`;
-      document.location.href = url_redirection_cas;
+      if (!this._authService.isAuthenticated()) {
+        // TODO: set the local storage item 'expires_at' in the API route "gn_auth/login_cas"
+        localStorage.setItem('expires_at', moment().add(1, 'days').toISOString());
+        document.location.href = url_redirection_cas;
```

Comment:
**Suggestion:** It's not safe to set the 'expires_at' value directly in the client-side code. This value should be set on the server-side when the token is issued, to prevent potential manipulation of the expiry time on the client-side. [security]
```suggestion
      if (!this._authService.isAuthenticated()) {
          document.location.href = url_redirection_cas;
      }
```